core/src/test/java/io/undertow/server/handlers/PredicatedHandlersTestCase.java

```diff
@@ -159,4 +159,86 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
         handler.handleRequest(e);
         Assert.assertEquals("foo'bar",e.getResponseHeaders().get("test-header").getFirst());
     }
+
+    @Test
+    public void testRewriteWithLineBreaks() throws IOException {
+        DefaultServer.setRootHandler(
+                Handlers.predicates(
+
+                        PredicatedHandlersParser.parse(
+                                        "path(/skipallrules)\n and true -> done\n" +
```

Comment:
Please adjust the test to include multiple line breaks in combination with carriage returns and tabs in a row.

Reply:
Closing requirement.